go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/abbot/go-http-auth v0.4.0
 	github.com/andybalholm/brotli v1.1.0
 	github.com/aryszka/jobqueue v0.0.3
+	github.com/benburkert/pbench v0.0.0-20160623210926-4ec5821845ef
```

Comment:
Linking since will be helpful here when trying out with go 1.24, https://github.com/benburkert/pbench/issues/2

Reply:
I don't like that we pull in a dependency that was last updated 9 years ago TBH.

Reply:
After reading the code I am not even sure it does the right thing, e.g. I don't see how it accounts for warmup benchmark runs.